portfolio.html

```diff
@@ -27,23 +28,58 @@ <h2>About me</h2>
 		<p class="paragraph">I am a recent graduate of IO academy, eager and motivated to throw myself into work. My degree and this course have given me technical skills in online web creation, video production and coding, whilst allowing me to lead and manage multiple group projects at once. I am reliable, confident and always keen to contribute as part of a team. I am dedicated and committed to any task and never afraid to step forward and lead.</p>
 	</section>
 	<section id="portfolio">
-		<div class="project-box">
-			<div class="project"></div>
-			<div class="content"></div>
-			<div class="description"></div>
+		<h2>Portfolio</h2>
+		<p>This is an short ensamble of the work I have done at IO</p>
+		<div class="project">
+			<h3>Project name</h3>
+			<div class="content">
+				<a href="#">
+					<img class="project-image" src="placeholder.jpg" alt="project placeholder">
+				</a>
+				<div class="description">
+					<p>This is a project I undertook at the IO academy. I did a little bit of this, a little bit of that, sprinlked a little shiraz and a little bit of flair and this sweet ride is the finished product. I excelled in this, needed to improve on that. If you click the picture you can see my push and pulls and my repo's.</p>
+				</div>
+			</div>
 		</div>
-		<div class="project-box">
-			<div class="project"></div>
-			<div class="content"></div>
-			<div class="description"></div>
+		<div class="project">
```

Comment:
"project_container" or similar would be better name - hard to tell what it is related to the project section